### PR TITLE
Moved cniPlugin parameter under access/core sections

### PR DESCRIPTION
**Pull request by sureshmarikkannu**

### CODE REVIEW

File: roles/core/templates/sdcore-5g-values.yaml [hunk 1]
```diff
@@ -314,18 +314,19 @@ omec-user-plane:
         enabled: false    # SRIOV is disabled by default
       hugepage:
         enabled: false    # Should be enabled if DPDK is enabled
-      cniPlugin: macvlan  # Can be any other plugin. Dictates how IP address are assigned
```
Comment (gab-arrobo) [inline roles/core/templates/sdcore-5g-values.yaml:317]: I think the `ipam` parameter also has to go inside each interface.

Comment (sureshmarikkannu) [reply roles/core/templates/sdcore-5g-values.yaml:317]: done

**Review by gab-arrobo — APPROVED** [review]
+1

**Comment by gab-arrobo** [comment]
@llpeterson please feel free to merge thos PR when you have a chance. Thanks!

**Comment by llpeterson** [comment]
Thanks for the fix.

**Pull request merged**